src/hooks/useNFTsSortBy.ts

```diff
@@ -20,7 +20,9 @@ export function useNftSort() {
   const [sortBy, sortDirection] = parseNftSort(nftSortData);
 
   return {
-    updateNFTSort: (nftSort: NftSort) => setNftSortData(nftSort),
+    updateNFTSort: (nftSort: NftSort) => {
```

Comment:
can be follow up: this whole file can be deleted

Reply:
done